### PR TITLE
Handle jnlp filename with space for shortcuts

### PR DESCRIPTION
**Pull request by sclassen**

### CODE REVIEW

**Comment by sclassen** [comment]
I will replace the constants with "\"" during backporting

Am 18. April 2019 14:20:52 MESZ schrieb Philippe Doussot <notifications@github.com>:

**Comment by hendrikebbers** [comment]
> I finally dont know if using common/src/main/java/net/adoptopenjdk/icedteaweb/IcedTeaWebConstants.java
> to define DOUBLE_QUOTE is a good idea beacause IcedTeaWebConstants is not present in 1.8 branch
We created IcedTeaWebConstants in master to define exactly such global constants that are used at several places in ITW. Sure it is not part of 1.8 and 1.7 but this is no reason for me to not use it. Otherwise we could not create new classes in the master ;)

File: core/src/main/java/net/sourceforge/jnlp/util/WindowsDesktopEntry.java [hunk 1]
```diff
@@ -94,9 +97,9 @@ public void createWindowsMenu() throws IOException {
             menuDir.mkdir();
         }
         final String JavaWsBin = XDesktopEntry.getJavaWsBin();
-        final ShellLink sl = ShellLink.createLink(JavaWsBin).setCMDArgs(file.getSourceLocation().toString());
+        final ShellLink sl = ShellLink.createLink(JavaWsBin).setCMDArgs(quoted(file.getSourceLocation()));
         // setup uninstall shortcut
-        final ShellLink ul = ShellLink.createLink(JavaWsBin).setCMDArgs("-Xclearcache " + file.getFileLocation().toString());
+        final ShellLink ul = ShellLink.createLink(JavaWsBin).setCMDArgs("-Xclearcache " + quoted(file.getFileLocation()));
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/util/WindowsDesktopEntry.java]: `-Xclearcache` as const

Comment (sclassen) [reply core/src/main/java/net/sourceforge/jnlp/util/WindowsDesktopEntry.java]: there is much more to clean up in this class.
Will postpone to not overload this PR

**Comment by sclassen** [comment]
@douph1 I removed the unquoting as you did in the 1.8 branch (#180)
Thanks for the work

**Pull request merged**